src/ua/griddynamics/Main.java

```diff
@@ -0,0 +1,127 @@
+package ua.griddynamics;
+
+import java.io.File;
+import java.util.InputMismatchException;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Main {
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        boolean quit = false;
+        final User USER = new User();
+        while (!quit) {
+            Menu.menuMain();
+            int choice = SCANNER.nextInt();
+            SCANNER.nextLine();
+            switch (choice) {
+                case 1:
+                    System.out.println("\nEnter income:");
+                    double income = SCANNER.nextDouble();
+                    SCANNER.nextLine();
+                    USER.addIncome(income, USER);
+                    break;
+                case 2:
+                    addPurchase(USER);
+                    Purchase.savePurchases(USER);
+                    break;
+                case 3:
+                    do {
```

Comment:
it is better to use while where it is possible (not do while)

Reply:
Done